### ISSUE TITLE
Need to support custom properties in ConvertTo-WPFGrid

### ISSUE THREAD

**Issue by jdhitsolutions** [jdhitsolutions/PSScriptTools#27]
```powershell
PS C:\> Get-process | sort ws | 
select -first 10 -Property ID,Name,WS,VM,NPM,Starttime,@{Name="runtime";Expression={(get-date) - $_.starttime }} | 
ConvertTo-WPFGrid
Exception calling "Substring" with "2" argument(s): "Length cannot be less than zero.
Parameter name: length"
At C:\Program Files\WindowsPowerShell\Modules\PSScriptTools\2.1.0\functions\ConvertTo-WPFGrid.ps1:164 char:9
+         $cmd = [scriptblock]::Create($myinvocation.line.substring(0,  ...
+         ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
    + CategoryInfo          : NotSpecified: (:) [], MethodInvocationException
    + FullyQualifiedErrorId : ArgumentOutOfRangeException
```

**Comment by jdhitsolutions** [comment]
The problem is not the hashtable.  The entire command needs to be a single pipelined expression. Need to better handle the error.

**Comment by jdhitsolutions** [comment]
Fixed in v2.2.0
